doc/integrations/redshift.ipynb

```diff
@@ -0,0 +1,1182 @@
+      ],
+      "text/plain": [
+       "++\n",
+       "||\n",
+       "++\n",
+       "++"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%sql\n",
+    "DROP TABLE taxi;\n",
+    "\n",
+    "CREATE TABLE taxi  (\n",
+    "    VendorID                BIGINT,\n",
+    "    tpep_pickup_datetime    TIMESTAMP,\n",
+    "    tpep_dropoff_datetime   TIMESTAMP,\n",
+    "    passenger_count         DOUBLE PRECISION,\n",
+    "    trip_distance           DOUBLE PRECISION,\n",
+    "    RatecodeID              DOUBLE PRECISION,\n",
+    "    store_and_fwd_flag      VARCHAR(1),\n",
+    "    PULocationID            BIGINT,\n",
+    "    DOLocationID            BIGINT,\n",
+    "    payment_type            BIGINT,\n",
+    "    fare_amount             DOUBLE PRECISION,\n",
+    "    extra                   DOUBLE PRECISION,\n",
+    "    mta_tax                 DOUBLE PRECISION,\n",
+    "    tip_amount              DOUBLE PRECISION,\n",
+    "    tolls_amount            DOUBLE PRECISION,\n",
+    "    improvement_surcharge   DOUBLE PRECISION,\n",
+    "    total_amount            DOUBLE PRECISION,\n",
+    "    congestion_surcharge    DOUBLE PRECISION,\n",
+    "    airport_fee             DOUBLE PRECISION\n",
+    ");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29eaffd7-49f9-4a78-8347-88001eefcf49",
+   "metadata": {},
+   "source": [
+    "Now, we use `COPY` to copy a `.parquet` file stored in an S3 bucket:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "104cca47-2474-47aa-b7a5-ab358b1097d4",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<span style=\"None\">Running query in &#x27;redshift-sqlalchemy&#x27;</span>"
+      ],
+      "text/plain": [
+       "Running query in 'redshift-sqlalchemy'"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<table>\n",
+       "    <thead>\n",
+       "        <tr>\n",
+       "        </tr>\n",
+       "    </thead>\n",
+       "    <tbody>\n",
+       "    </tbody>\n",
+       "</table>\n",
+       "<span style='font-style:italic;font-size:11px'><code>ResultSet</code> : to convert to pandas, call <a href='https://jupysql.ploomber.io/en/latest/integrations/pandas.html'><code>.DataFrame()</code></a> or to polars, call <a href='https://jupysql.ploomber.io/en/latest/integrations/polars.html'><code>.PolarsDataFrame()</code></a></span><br>"
+      ],
+      "text/plain": [
+       "++\n",
+       "||\n",
+       "++\n",
+       "++"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%sql\n",
+    "COPY taxi\n",
+    "FROM 's3:///some-bucket/yellow_tripdata_2023-01.parquet'\n",
```

Comment:
> This cell wouldn't work by default like the rest of our integration tests,

I'm assuming you meant "tutorials", not "integration tests". what do you mean by "like the rest of"?

> either use a local file or put some comments in the beginning to load that file to the expected S3 location and add a link on how to create the IAM role.

I don't think COPY supports uploading from a local file, but I can add all the commands I ran to make this work (create bucket, upload data, configure permissions, etc.). I didn't add them initially because I think the majority of users that read this will already have *some data* in their redshift cluster, and I'd rather keep this brief. wdyt?

Reply:
ok, I added a dropdown with the commands they need to upload the parquet file